src/esp/bindings/MetadataMediatorBindings.cpp

```diff
@@ -25,6 +25,15 @@ void initMetadataMediatorBindings(py::module& m) {
           R"(The currently active dataset being used.  Will attempt to load
             configuration files specified if does not already exist.)")
 
+      /* --- Methods --- */
+      .def(
+          "dataset_exists", &MetadataMediator::sceneDatasetExists,
+          R"(Checks whether the passed name references an existing scene dataset)",
+          "dataset_name"_a)
+      .def(
+          "remove_dataset", &MetadataMediator::removeSceneDataset,
+          R"(Remove the given dataset from MetadataMediator.  If specified dataset is currently active, this will fail.)",
```

Comment:
Fail how? return false, throw an exception?

Reply:
Returns false, with a message.

Reply:
... and return false. Otherwise true. @jturner65 Is this actually a good candidate for an ESP_CHECK?

Reply:
I don't think so.  This is just unloading a particular dataset from MetadataMediator, and returning a success code reflecting whether the deletion happened or not.  This does not verify the existence of a particular disk resource, nor is it intended to do any kind of file management on disk, which is what I would think an ESP_CHECK would be for.